upgrading/topics/keycloak/changes.adoc

```diff
@@ -2,6 +2,18 @@
 
 === Migrating to 12.0.0
 
+==== Upgrade to Wildfly 21
+
+The {project_name} server was upgraded to use Wildfly 21 as the underlying container. This does not directly involve any
+specific {project_name} server functionality, but there are few changes related to the migration, which are worth mentioning.
+
+Dependency updates::
+  The dependencies were updated to the versions used by Wildfly 21 server. For example, Infinispan is now 11.0.4.Final.
+
+Configuration changes::
+  There are few configuration changes in the `standalone(-ha).xml` and `domain.xml` files. You should follow the <<_install_new_version>>
```

Comment:
Generally avoid "there are" which makes no sense grammatically, because "there is not the subject of the sentence."
You can usually rephrase using the verb "exists"
So please make it "A few configuration changes exist in..."

Reply:
Thanks for the suggestion, fixed in latest version.